application/src/main/java/org/opentripplanner/gtfs/graphbuilder/GtfsModule.java

```diff
@@ -360,51 +360,6 @@ private GtfsMutableRelationalDao loadBundle(GtfsBundle gtfsBundle) throws IOExce
       }
     }
 
-    for (ShapePoint shapePoint : store.getAllEntitiesForType(ShapePoint.class)) {
```

Comment:
@binh-dam-ibigroup You asked about this in a previous PR and all of this ugly code is now going away completely.